src/gui/creds/oauth.cpp

```diff
@@ -79,21 +84,44 @@ void OAuth::start()
                 req.setHeader(QNetworkRequest::ContentTypeHeader, "application/x-www-form-urlencoded");
                 auto reply = _account->sendRequest("POST", requestToken, req);
                 QTimer::singleShot(30 * 1000, reply, &QNetworkReply::abort);
-                QObject::connect(reply, &QNetworkReply::finished, this, [this, reply] {
+                QObject::connect(reply, &QNetworkReply::finished, this, [this, reply, socket] {
                     auto jsonData = reply->readAll();
                     QJsonParseError jsonParseError;
                     QJsonObject json = QJsonDocument::fromJson(jsonData, &jsonParseError).object();
                     QString accessToken = json["access_token"].toString();
                     QString refreshToken = json["refresh_token"].toString();
                     QString user = json["user_id"].toString();
+                    QUrl messageUrl = json["message_url"].toString();
 
                     if (reply->error() != QNetworkReply::NoError || jsonParseError.error != QJsonParseError::NoError
                         || json.isEmpty() || refreshToken.isEmpty() || accessToken.isEmpty()
                         || json["token_type"].toString() != QLatin1String("Bearer")) {
-                        qCWarning(lcOauth) << "Error when getting the accessToken" << reply->error() << json << jsonParseError.errorString();
+                        QString errorReason;
+                        QString errorFromJson = json["error"].toString();
+                        if (!errorFromJson.isEmpty()) {
+                            errorReason = tr("Error returned from the server: <em>%1</em>")
```

Comment:
Yes, I did not add the <br> here because the error returned from the server is usually just one word. (very short)
The other errors are centences